pip_check_reqs/find_extra_reqs.py

```diff
@@ -53,7 +53,7 @@ def find_extra_reqs(
             item_location_rel = Path(package_location) / item
             item_location = common.cached_resolve_path(path=item_location_rel)
             try:
-                relative_item_location = item_location.relative_to(here)
+                relative_item_location = item_location.relative_to(package_location)
```

Comment:
This unfortunately means that you are ending up with `package.files == package_files`, which seems wholly redundant.